starter-code/js/index.js

```diff
@@ -1,23 +1,79 @@
-function deleteItem(e){
 
-}
+var shoppingCart = []
 
-function getPriceByProduct(itemNode){
 
+function Product(name, price) {
+  this.name = '';
+  this.quantity;
+  this.price;
 }
 
-function updatePriceByProduct(productPrice, index){
 
+function getTotalPrice(product) {
+  var productCount = document.querySelectorAll(".wrapper");
+  for (var i=0; i<productCount.length; i++) {
+    var price = Number(document.getElementsByClassName("price")[i].innerText);
+    var quantity = Number(document.querySelectorAll(".quantity")[i].value);
+    var result = price * quantity;
+    document.getElementsByClassName("total")[i].innerText = result;
+  } 
+  var price = document.querySelectorAll('.total');
+  var total = 0;
+  price.forEach(function(item) {
+    total += Number(item.innerText);
+  });
+  var printResult = document.getElementById("result");
+  printResult.innerText = total;
 }
 
-function getTotalPrice() {
 
+
+function deleteItem(e){
+  e.currentTarget.parentNode.parentNode.remove();
```

Comment:
Maybe we can call `getTotalPrice()` for calculate the new price when we delete the item.